Makefile

```diff
@@ -10,53 +10,60 @@ endif
 
 .PHONY: install
 install: ## make install # Install roles dependencies
-	@ansible-galaxy install --roles-path vendor/ -r requirements.yml
```

Comment:
I think it is a bit to complicated to look for the roles_path option.
You have to check to many places to be sure a roles_path option is set.
* ANSIBLE_CONFIG (an environment variable)
* ansible.cfg (in the current directory)
* .ansible.cfg (in the home directory)
* /etc/ansible/ansible.cfg

I think the simplest way is to set a parameter if you like to change the default behaviour.
The default install path for Ansible galaxy roles is "/etc/ansible/roles".
So I think the directory "roles" should be the default that works out of the box.
`roles_path ?= "roles/"`
...
`@ansible-galaxy install --roles-path="$(roles_path)" --role-file="requirements.yml"`

`make install`
Install Ansible galaxy roles inside directory "roles"

`make roles_path=vendor install`
Install Ansible galaxy roles inside directory "vendor"

Do you agree with this implementation suggestion?